Makefile

```diff
@@ -91,11 +91,12 @@ TARGETS += vmtoolsd-guest-agent
 TARGETS += wasmedge
 TARGETS += xen-guest-agent
 TARGETS += zfs
+TARGETS += dvb-si2168-firmware
```

Comment:
this file is auto-generated from `.kres.yaml` via `make rekres`

Reply:
After running said `mak rekres`, the makefile was changed to a format which makes make not work.

```
-define HELP_MENU_HEADER
+export define HELP_MENU_HEADER
 # Getting Started

 To build this project, you must have the following installed:
@@ -144,8 +143,6 @@ respectively.

 endef

-export HELP_MENU_HEADER
-
```


```
$ make rekres
Makefile:103: *** missing separator.  Stop.
```

Reply:
Was running GNU Make 3.81, despite their being newer GNU Make 4.4.1.